modular_skyrat/modules/modular_implants/code/soulcatcher/soulcatcher_component.dm

```diff
@@ -339,6 +345,52 @@ GLOBAL_LIST_EMPTY(soulcatchers)
 	room_to_join.add_soul_from_ghost(src)
 	return TRUE
 
+/**
+ * Args:
+ * datum/component/soulcatcher/catcher - The soulcatcher we are checking.
+ *
+ * Returns: TRUE by default.
+ */
+/mob/proc/can_join_soulcatcher(datum/component/soulcatcher/catcher)
+	return TRUE
+
+/**
+ * Args:
+ * datum/component/soulcatcher/catcher - The soulcatcher we are checking.
+ *
+ * Returns: False if super returned false, false if the catcher is not joinable by ghosts, true otherwise.
+ */
+/mob/dead/observer/can_join_soulcatcher(datum/component/soulcatcher/catcher)
+	. = ..()
+
+	if (!.)
+		return
+
+	if (!catcher.ghost_joinable)
+		return FALSE
+
```

Comment:
you need a `return TRUE` here, otherwise the proc is always going to return False.

Reply:
Wrong, super returns true by default, Ill comment it for clarity

Reply:
(and we call super!)

Reply:
(it isn't super, it's the parent, there is no super in DM :trollface:)